src/Framework/TestCase.php

```diff
@@ -1942,89 +1946,100 @@ private function checkRequirements(): void
 
     private function handleDependencies(): bool
     {
-        if (!empty($this->dependencies) && !$this->inIsolation) {
-            $className  = \get_class($this);
-            $passed     = $this->result->passed();
-            $passedKeys = \array_keys($passed);
+        if (empty($this->dependencies) || $this->inIsolation) {
+            return true;
+        }
 
-            foreach ($passedKeys as $key => $value) {
-                $pos = \strpos($value, ' with data set');
+        $className  = \get_class($this);
```

Comment:
```suggestion
```

Variable is unused.

Reply:
fixed; however no idea how this came to be